include/RAJA/config.hpp.in

```diff
@@ -196,17 +196,19 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 #if defined(RAJA_ENABLE_CUDA)
 #define RAJA_ALIGN_DATA(d)
 #else
-
-#if __ICC < 1300  // use alignment intrinsic
 #define RAJA_ALIGN_DATA(d) __assume_aligned(d, DATA_ALIGN)
-#else
-#define RAJA_ALIGN_DATA(d)  // TODO: Define this...
 #endif
 
+#if defined(_OPENMP) && (_OPENMP >= 201307)
+#define RAJA_SIMD  _Pragma("omp simd") \
+                   _Pragma("vector aligned")
```

Comment:
Why is there a "pragma vector aligned" on an OpenMP branch? These are not provided or supported as part of OpenMP.

Reply:
Ah thanks Tom. As a consequence I believe the compiler was ignoring the omp simd pragma. Addressing this resulted in several test failing... Namely the test-nested and test-reduce.

Reply:
@artv3 we are using the macro constant RAJA_ENABLE_OPENMP throughout the code to guard openmp sections, please use that wherever you can.